src/apps/frontend/pages/authentication/otp/otp-form-hook.ts

```diff
@@ -0,0 +1,65 @@
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+
+import { useAuthContext } from '../../../contexts';
+import { AsyncError } from '../../../types';
+
+interface OTPFormProps {
+  onError: (err: AsyncError) => void;
+  onResendSuccess: () => void;
+  onSuccess: () => void;
+}
+const useOTPForm = ({
+  onError, onResendSuccess, onSuccess,
+}: OTPFormProps) => {
+  const {
+    isVerifyOTPLoading, verifyOTPError, verifyOTPResult, sendOTP, verifyOTP,
+  } = useAuthContext();
+
+  const searchParams = new URLSearchParams(window.location.search);
+  const phoneNumber = searchParams.get('ph');
```

Comment:
Avoid using `ph` it makes code hard to read. Its ok to pass `phone_number` in QS